torch/onnx/_internal/fx/dynamo_graph_extractor.py

```diff
@@ -194,6 +194,8 @@ def generate_fx(
             *model_args,
             tracing_mode=fx_mode,
             fake_mode=fake_mode,  # type: ignore[arg-type]
+            aten_graph=fake_mode
```

Comment:
This is very high priority. Let's call soon.

Reply:
Sure. This is needed because of the integration between fake tensor and dynamic shape. Tracking through https://github.com/pytorch/pytorch/issues/105467 for now